ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-release-4.15.yaml

```diff
@@ -11,14 +11,14 @@ base_images:
     name: hypershift-operator
     namespace: hypershift
     tag: latest
-  hypershift-tests:
-    name: hypershift-tests
-    namespace: hypershift
-    tag: latest
   openshift_release_golang-1.20:
     name: release
     namespace: openshift
     tag: golang-1.20
+  test-bin:
```

Comment:
maybe try `hypershift-tests` instead of `test-bin`.
this is the name used in the hypershift release branches.